protocols/imap/src/main/java/org/apache/james/imap/decode/ImapRequestStreamLineReader.java

```diff
@@ -102,4 +102,9 @@ protected void commandContinuationRequest() throws DecodingException {
         }
     }
 
+    @Override
+    public void close() throws IOException {
+        input.close();
```

Comment:
what if this line throws?

Reply:
the caller deals with it (and ignore it currently)

Reply:
but the next line never execute